From 4dd30c5e1b29a8044084226bad7b69f7702fde4f Mon Sep 17 00:00:00 2001
From: sahu39 <mr.sahu39@gmail.com>
Date: Wed, 12 Dec 2018 08:48:17 +0000
Subject: [PATCH 1/2] prog2.py is added 1stbranch

---
 prog2.py | 1 +
 1 file changed, 1 insertion(+)
 create mode 100644 prog2.py

diff --git a/prog2.py b/prog2.py
new file mode 100644
index 0000000..01466c9
--- /dev/null
+++ b/prog2.py
@@ -0,0 +1 @@
+print("Hello prog2.py is added in the python examples and in 1stbranch & from pythonanywhere.com local repository ")
-- 
1.9.1

